### PR TITLE
Feature: set glencoe caching to False for prod environments

### PR DESCRIPTION
**Pull request by lsh-0**
- [x] blocked by: https://github.com/elifesciences/builder-configuration/pull/75

### CODE REVIEW

File: salt/lax/config/opt-bot-lax-adaptor-app.cfg [hunk 1]
```diff
@@ -20,6 +25,9 @@ cdn_iiif: {{ scheme }}://{{ pillar.elife.env }}--cdn-iiif.elifesciences.org/lax:
 {%- endif %}
 iiif: {{ scheme }}://{{ pillar.elife.env }}--iiif.elifesciences.org/lax:
 
+[glencoe]
+cache_requests: {{ pillar.elife.env not in ['prod', 'continuumtest'] }}
```
Comment (giorgiosironi) [inline salt/lax/config/opt-bot-lax-adaptor-app.cfg]: `end2end` should be included with those two

Comment (giorgiosironi) [reply salt/lax/config/opt-bot-lax-adaptor-app.cfg]: shouldn't this be a `pillar.lax.glencoe.cache_requests` value?

Comment (lsh-0) [reply salt/lax/config/opt-bot-lax-adaptor-app.cfg]: I can make it configurable through pillar data

Comment (lsh-0) [reply salt/lax/config/opt-bot-lax-adaptor-app.cfg]: updated. builder-config PR: https://github.com/elifesciences/builder-configuration/pull/75

**Comment by lsh-0** [comment]
> (pending green build due to builder-configuration merge)

merged, kicking build

**Pull request merged**